client/src/components/Donation/DonateForm.js

```diff
@@ -140,10 +189,16 @@ class DonateForm extends Component {
     this.setState({ donationAmount });
   }
 
-  async handleStripeCheckoutRedirect(e) {
-    const { stripe } = this.props;
+  async handleStripeCheckoutRedirect(e, paymentMethod) {
+    const { stripe } = this.state;
```

Comment:
What happens if stripe has not loaded?  Can we disable the buttons until that has happened?

Reply:
We handle that logic in componentDidMount. We could ultimately display a loader similar to paypal until stripe loads.

Reply:
There is another pr coming up that combines all of those stripe buttons into one. We can address this issue there.

Reply:
Okay, makes sense.  If the button isn't around to be clicked before Stripe has loaded, this seems fine.